containers/kubernetes/scripts/install-rpms.sh

```diff
@@ -29,22 +29,20 @@ while [ $# -gt 0 ]; do
 	shift
 done
 
-if [ -z "$COCKPIT_RPM_URL" ]; then
-    COCKPIT_RPM_URL="https://kojipkgs.fedoraproject.org/packages/cockpit"
-fi
-
 if [ -z "$INSTALLER" ]; then
     INSTALLER="dnf"
 fi
 
-if [ -z "$OS" ]; then
-    OS=$(rpm -q --qf "%{release}" basesystem | sed -n -e 's/^[0-9]*\.\(\S\+\).*/\1/p')
-fi
+OSVER=$(. /etc/os-release && echo "$VERSION_ID")
 
 for package in $@
 do
-    rpm=$(ls /container/rpms/$package-*.$arch.rpm || true)
-    if [ -z "$rpm" ] && [ -n "$USE_REPO" ]; then
+    rpm=$(ls /container/rpms/$package-*$OSVER.*$arch.rpm || true)
```

Comment:
This isn't new, but can we be sure that we only use this for arch specific packages, not `-noarch` ones?